state_names.py

```diff
@@ -3,18 +3,23 @@
 State names in a dictionary
 File needs reformatting
 """
+
 # TODO: Reformat this file so the dictionary code follows PEP 8 convention
 CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
                 "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
 print(CODE_TO_NAME)
-for key in CODE_TO_NAME:
```

Comment:
the for loop prints out all the code represents place

Reply:
yes the red blocks of code in not in the actua code